tracer/src/Datadog.Trace/Logging/DirectSubmission/Sink/PeriodicBatching/BoundedConcurrentQueue.cs

```diff
@@ -49,6 +49,10 @@ public BoundedConcurrentQueue(int? queueLimit = null)
         // Internal for testing
         internal ConcurrentQueue<T> InnerQueue => _queue;
 
+        public int Count => _queue.Count;
+
+        public bool IsEmpty => _queue.IsEmpty;
+
```

Comment:
I've just twigged - if you're adding values here, you're probably reusing this for something _other_ than DirectLogSubmission. If so, I think we should either:

1. Move this out of the "logging" subfolder and _potentially_ add more tests around it. We already tore it apart compared to the original, and IIRC there were some minor concerns about it, so may warrant another look.
2. Create a separate implementation for debugging

Reply:
Sounds good.
I've moved `BoundedConcurrentQueue` from logging namespace to `Utils` at 9495b0bebfb02c5374d9b4e6c82004fb22085b13 and added some really simple tests for added properties. 
Please suggest more tests, if you have some in mind.

Reply:
Oh, I see.
I agree with you, I prefer not use them as well, but for I cannot see, how can I achieve same functionality, without using them 🤔 
May be by exposing inner queue 🤔 
But this kind of pattern breaker 😒 

We can change it in future, if we see any issues with that.

Reply:
I was just trying to double check, (it's tricky to track it's usages until after this PR is merged), but it looks like there's already a lot of incorrect patterns around this 😬 it's hard to trace through on GitHub though... I feel like most of the usages should be refactored, but again, we can/should push that to a follow up PR IMO

Reply:
OK, it will be dome in follow up PR.
Resolving for now.